packages/mimecast/data_stream/audit_events/_dev/test/pipeline/test-audit-events.log-expected.json

```
@@ -203,9 +204,10 @@
                 "preserve_original_event"
             ],
             "mimecast": {
-                "application": "Administration Console, Method: Office 365, 2FA: TOTP",
+                "application": "Administration Console",
                 "category": "authentication_logs",
-                "eventInfo": "Intermediate authentication for johndoe@example.com \u003cJohn Doe\u003e, Date: 2021-10-11, Time: 18:17:26 BST, IP: 67.43.156.15, Application: Administration Console, Method: Office 365, 2FA: TOTP"
+                "eventInfo": "Intermediate authentication for johndoe@example.com \u003cJohn Doe\u003e, Date: 2021-10-11, Time: 18:17:26 BST, IP: 67.43.156.15, Application: Administration Console, Method: Office 365, 2FA: TOTP",
```

```suggestion
                "eventInfo": "Intermediate authentication for johndoe@example.com \u003cJohn Doe\u003e, Date: 2021-10-11, Time: 18:17:26 BST, IP: 67.43.156.15, Application: Administration Console, Method: Office 365, 2FA: TOTP",
```

Could we also keep the 2FA field? Just adding the mapping and not discarding it in the pipeline.

---

Yes, changes implemented. 2FA will remain in json as  a new property.